### ISSUE TITLE
Odd flag/language behaviour when updating from beta to v0.1.0

### ISSUE THREAD

**Issue by ehowey** [sanity-io/document-internationalization#32]
Hi All!

Great work on this! Loving it.  The update to v0.1.0 went really smooth with one small hiccup.

Here is what my translations selector looked like before during beta versions:
<img width="792" alt="Screen Shot 2022-03-12 at 8 31 27 PM" src="https://user-images.githubusercontent.com/204841/158043830-ab373db6-f361-4482-84ab-a7051fcd7eca.png">

And here is what it looks like now with v0.1.0:
<img width="362" alt="Screen Shot 2022-03-12 at 8 30 19 PM" src="https://user-images.githubusercontent.com/204841/158043834-7a9c31a8-9091-43c9-96a7-2ae7ca65d8db.png">

I prefer the 2nd version, with the caveat that it does not seem to be properly recognizing the ids and languages for the flags and I could see this leading to some confusion for editors.

Here is what my `supportLanguages.js` config file looks like:

```js
export const supportedLanguages = [
  { id: "en", title: "English", isDefault: true },
  { id: "fr", title: "French" },
  { id: "en_ca", title: "English (CA)" },
  { id: "fr_ca", title: "French (CA)" },
  { id: "fr_fr", title: "French (FR)" },
];

export const baseLanguage = supportedLanguages.find((l) => l.isDefault);
```
EDIT:

I am going to leave this issue up but the more I think about it, maybe it is better off for my content editors to not have flags at all.  It is a bit confusing, as flags represent countries and not languages. I can see in the docs that I can workaround and export my own flag component and just return null so the flag area is always blank, which I think makes more sense. It doesn't seem to work right now as I think the part that needs to implemented has changed with the new language selector.

**Comment by LiamMartens** [comment]
@ehowey This is now resolved in 0.1.1! I also re-enabled the custom flag implementation - you were right this got lost in the new UI

**Comment by ehowey** [comment]
Wow - that was fast. Amazing!
